go.mod

```diff
@@ -115,3 +115,5 @@ require (
 	sigs.k8s.io/kube-storage-version-migrator v0.0.6-0.20230721195810-5c8923c5ff96 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.3.0 // indirect
 )
+
+replace github.com/openshift/api => /root/projects/api
```

Comment:
Can we get this to a fixed commit in some repository rather than to a local folder

Reply:
This is for creating private build for QE testing. I'll bump the api version once the API PR is merged.